app/views/directives/osc-secrets.html

```diff
@@ -0,0 +1,34 @@
+<ng-form name="secretsForm" class="osc-secrets-form">
+  <div class="form-group">
+
+    <label class="picker-label">{{displayType | startCase}} Secret</label>
+    <ui-select ng-model="pickedSecret.name">
+      <ui-select-match placeholder="Secret name">{{$select.selected}}</ui-select-match>
+      <ui-select-choices repeat="secret in (secretsByType[type] | filter : $select.search)">
+        <div ng-bind-html="secret | highlight : $select.search"></div>
+      </ui-select-choices>
+    </ui-select>
+    <div ng-switch="displayType">
+      <div class="help-block" ng-switch-when="source">
+        Secret with credentials for pulling your source code.
+        <a href="{{'git_secret' | helpLink}}" aria-hidden="true" target="_blank"><span class="learn-more-inline">Learn more<i class="fa fa-external-link"></i></span></a>
+      </div>
+      <div class="help-block" ng-switch-when="pull">
+        Secret for authentication when pulling images from a secured registry.
+        <a href="{{'pull_secret' | helpLink}}" aria-hidden="true" target="_blank"><span class="learn-more-inline">Learn more<i class="fa fa-external-link"></i></span></a>
+      </div>
+      <div class="help-block" ng-switch-when="push">
+        Secret for authentication when pushing images to a secured registry.
+        <a href="{{'pull_secret' | helpLink}}" aria-hidden="true" target="_blank"><span class="learn-more-inline">Learn more<i class="fa fa-external-link"></i></span></a>
+      </div>
```

Comment:
Remove `aria-hidden` from these 3 help links, but add them to the icon `i` element.